app/views/helpers/injected/RecruitmentBannerHelper.scala.html

```diff
@@ -0,0 +1,34 @@
+@*
+* Copyright 2021 HM Revenue & Customs
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
+
+@import config.FrontendAppConfig
+
+@this()
+@()(implicit messages: Messages, appConfig: FrontendAppConfig)
+
+<div class="js-hidden notice-banner" id="notice-banner">
+  <div class="centered-content">
+    <div class="notice-banner__wrapper grid-layout">
+      <div class="grid-layout__column--2-3">
+        <h3 id="recruitment-banner-text" class="notice-banner__content">@Messages("banner.recruitment.text")</h3>
+        <a id="recruitment-banner-link" class="notice-banner__content" href="@appConfig.enterSurveyUrl">@Messages("banner.recruitment.link")</a>
+      </div>
+      <div class="grid-layout__column--1-3 text--right">
+        <a id="recruitment-banner-dismiss" class="notice-banner__close notice-banner__content" href="#">@Messages("banner.recruitment.dismiss")</a>
```

Comment:
Replace the calls to `object Messages` with the `@messages`
```suggestion
        <h3 id="recruitment-banner-text" class="notice-banner__content">@messages("banner.recruitment.text")</h3>
        <a id="recruitment-banner-link" class="notice-banner__content" href="@appConfig.enterSurveyUrl">@messages("banner.recruitment.link")</a>
      </div>
      <div class="grid-layout__column--1-3 text--right">
        <a id="recruitment-banner-dismiss" class="notice-banner__close notice-banner__content" href="#">@messages("banner.recruitment.dismiss")</a>
```